catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -64,6 +64,7 @@
         <bundle>mvn:ddf.catalog.core/catalog-core-resourcesizeplugin/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-camelcomponent/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-dynamic-registry/${project.version}</bundle>
+        <bundle>mvn:ddf.catalog.core/catalog-core-migratable/${project.version}</bundle>
```

Comment:
This is certainly an irremovable component of catalog-core; we will be building on this service as we continue to flesh out the migration system. Every DDF needs it to enable upgrade paths.